IsTheMartin/mrtnmrls
# Images is post are not being displayed

## Summary
Posts image placed on `/public/images/posts`

Images placed on `src/assets` are not generated dynamically for server configuration.

## Evidence

|Before|After|
|---|---|
|<img width="1435" alt="image" src="https://github.com/IsTheMartin/mrtnmrls/assets/38388605/268bfa1b-a73a-4f91-b1fb-c28e1d3b7dce">|<img width="1209" alt="image" src="https://github.com/IsTheMartin/mrtnmrls/assets/38388605/cf4887de-93c6-4d1f-95e9-07f98d0ea688">|

## Concern Information

To test please follow the next steps:

1. Navigate to any post with image
2. Observe image

## Issue ticket

- https://github.com/IsTheMartin/mrtnmrls/issues/30

## Check list

- [ x ] I did add reviewers to this PR
